integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -43,6 +43,10 @@
 {{- if .Values.jvmOptions }}
   {{- $alluxioJavaOpts = concat $alluxioJavaOpts .Values.jvmOptions }}
 {{- end }}
+{{- if .Values.metrics.enabled }}
+  {{- $metricsConfFile := "-Dalluxio.metrics.conf.file=/config/metrics.properties" }}
```

Comment:
I tested manually and this path seems to be wrong. When you mount the configmap `alluxio-metrics` to this path, it will be a directory:
```
[ec2-user@KubernetesSingleMasterLocal-masters-1 ~]$ kubectl exec -it alluxio-master-0 bash
Defaulting container name to alluxio-master.
Use 'kubectl describe pod/alluxio-master-0 -n default' to see all of the containers in this pod.
bash-4.4$ cd /config
bash-4.4$ ls -al -R
.:
total 0
drwxr-xr-x    3 root     root            32 Dec 16 08:38 .
drwxr-xr-x    1 root     root            65 Dec 16 08:38 ..
drwxrwsrwx    3 root     alluxio         85 Dec 16 08:38 metrics.properties
./metrics.properties:
total 0
drwxrwsrwx    3 root     alluxio         85 Dec 16 08:38 .
drwxr-xr-x    3 root     root            32 Dec 16 08:38 ..
drwxr-sr-x    2 root     alluxio         32 Dec 16 08:38 ..2020_12_16_08_38_53.343245412
lrwxrwxrwx    1 root     root            31 Dec 16 08:38 ..data -> ..2020_12_16_08_38_53.343245412
lrwxrwxrwx    1 root     root            25 Dec 16 08:38 metrics.properties -> ..data/metrics.properties
./metrics.properties/..2020_12_16_08_38_53.343245412:
total 4
drwxr-sr-x    2 root     alluxio         32 Dec 16 08:38 .
drwxrwsrwx    3 root     alluxio         85 Dec 16 08:38 ..
-rw-r--r--    1 root     alluxio        507 Dec 16 08:38 metrics.properties
```

Therefore, in order for this to work, this line should be `{{- $metricsConfFile := "-Dalluxio.metrics.conf.file=/config/metrics.properties/metrics.properties" }}`

Reply:
Could you change the configmap `alluxio-metrics` to be mounted to `/config/metrics` and make `-Dalluxio.metrics.conf.file=/config/metrics/metrics.properties`? Let's distinguish the directory name with the config file name.

Reply:
@jiacheliu3 , thanks for finding this out. Yes, totally make sense to differentiate the directory and then the config file name. This is fixed now.